build/yamls/patches/whereabouts/whereabouts.yml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: antrea-agent
+spec:
+  template:
+    spec:
+      initContainers:
+        - name: install-whereabouts-config
+          image: antrea
+          resources:
+            requests:
+              cpu: "100m"
+          command: ["install_whereabouts_config"]
+          securityContext:
+            capabilities:
+              add:
+                # SYS_MODULE is required to load the OVS kernel module.
+                - SYS_MODULE
```

Comment:
I don't think this applies to the `install-whereabouts-config` initContainer?

Reply:
yes. removed. thanks.